src/db/migrations/37-change-crdt-state-table-value-column-capacity.ts

```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex';
+
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.dropView(tableNameToViewName(Table.crdtState));
+  await knex.schema.alterTable(Table.crdtState, table => {
+    table.string('value', 1000).alter();
```

Comment:
oh nice catch. should this maybe be even longer/bigger? it looks like the biggest we have in practice that could potentially be exposed is 20000, so i think going with that should be fine